lib/iris/tests/unit/util/test_equalise_cubes.py

```diff
@@ -0,0 +1,291 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the BSD license.
+# See LICENSE in the root of the repository for full licensing details.
+"""Unit tests for the :func:`iris.util.equalise_cubes` function."""
+
+import warnings
+
+from cf_units import Unit
+import numpy as np
+from numpy.random import Generator
+import pytest
+
+from iris.coords import DimCoord
+from iris.cube import Cube
+from iris.util import equalise_cubes
+from iris.warnings import IrisUserWarning
+
+
+def _scramble(inputs, rng=95297):
```

Comment:
Changing a different seed should *not* invalidate the checks in fact, since mostly the only difference it makes is that where the tests construct the expected result metadata, they sometimes need to be re-sorted in the same way as the input cubes (like [this example](https://github.com/SciTools/iris/blob/62285b58e1986145215a731402eedcef90268575/lib/iris/tests/unit/util/test_equalise_cubes.py#L149-L150)).
The exception is [the unify-units test](https://github.com/SciTools/iris/blob/62285b58e1986145215a731402eedcef90268575/lib/iris/tests/unit/util/test_equalise_cubes.py#L245-L246), because it resolves to the first-encountered reference time, which obviously depends on the ordering. 

So, the "magic number" was in fact chosen so that the re-sorted order is non-trivial for all small numbers of entries (approx 2-10), in particular that neither the first or last entries should stay the same.  Though naturally that isn't terribly obvious !
I will add  a comment about it, while trying to keep it shorter than all this ...